src/plugins/chat/public/actions/navigate_action.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiPanel, EuiText, EuiIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { useAssistantAction } from '../../../context_provider/public';
+import { useOpenSearchDashboards } from '../../../opensearch_dashboards_react/public';
+import { CoreStart } from '../../../../core/public';
+
+interface NavigateArgs {
+  appId: string;
+  path?: string;
+  description?: string;
```

Comment:
navigation has two other optional properties, replace and state. state is used to pass context/data and it might not be the correct way for context sharing for chatbot. It is often used to share some app states which I don't think this tool should manage. for `replace`, it can control whether navigation creates a new history entry (push) or replaces the current one (replace) which is useful if we need to navigate back and force. let's add `replace?: boolean` to follow the navigation pattern.